app/assets/javascripts/application.js

```diff
@@ -14,4 +14,5 @@
 //= require turbolinks
 //= require 'govuk/all'
 //= require 'moj/all'
+//= require 'accessible-autocomplete.min'
```

Comment:
would be good to record the origin and version of this 3rd-party component in comments and/or a filename. We can use a non-minified version of such components, as the asset pipeline minifies things anyway, which is sometimes useful if a bugfix is released then it can be diffed against the original.

Reply:
Ideally we'd use the JavaScript package manager (NPM or Yarn) to handle this dependency, rather than checking the file in to our repo.

It looks like we have `package.json` and `yarn.lock` files already set up – do you know if the project is already using this?

Reply:
we do use NPM packages already for the GOV.UK styles, so I think it should be possible?

Reply:
The Sprockets setup isn't quite as slick as Webpacker – it looks like the JS module directory needs to be registered as an include path in `config/initializers/assets.rb`:

https://github.com/ministryofjustice/offender-management-allocation-manager/blob/e22f0e37c670cb3b3b8b375e9b34f2a6be36e0a6/config/initializers/assets.rb#L8-L15

The `node_modules` directory is already there, so that might already be enough.